### PR TITLE
[6840]Fix-pagination-buttons

### LINKED ISSUE #6840: Theme issues on 2.10 branch (master)
**CKAN version**
master

- [x] Pagination buttons are not styled

Screenshot 2022-05-03 at 12-39-24 Dataset - CKAN

- [x] Activity filter on Dashboard is floating on the left

Screenshot 2022-05-03 at 12-22-15 Manage - Adrià 333332 - Users - CKAN

- [x]  Activity icons not showing up (not sure if this is a font issue in my machine)

Screenshot 2022-05-03 at 14-59-04 Test datasets 222222 - CKAN

- [ ] DataTables sort buttons
Screenshot 2022-09-13 at 13-05-44 REREDS - 25_best_tacos_austin csv - CKAN

- [x] Clear upload button
Screenshot 2022-09-13 at 13-02-50 Edit - 25_best_tacos_austin csv - REREDS - CKAN

- [x] Spacing in custom extras fields
Screenshot 2022-09-13 at 11-38-30 Create Dataset - CKAN

- [x] License field margins and spacing
Screenshot 2022-09-13 at 11-38-22 Create Dataset - CKAN

### PR DESCRIPTION
**Pull request by TomeCirun**
Fixes #6840

### Features:

- [ ] includes tests covering changes
- [ ] includes updated documentation
- [ ] includes user-visible changes
- [ ] includes API changes
- [ ] includes bugfix for possible backport

Please [X] all the boxes above that apply

### CODE REVIEW

**Comment by smotornyuk** [comment]
@TomeCirun, theme CSS in CKAN is compiled from the sass-source. This means, that your change will disappear the next time someone recompiles the frontend assets.

Instead of direct changes to the CSS files, locate a problematic place in one of the SCSS files under `ckan/public/base/scss`, update it and recompile assets using `ckan sass` (you may need to install NodeJS deps via `npm ci`)

**Comment by TomeCirun** [comment]
@smotornyuk is this what you suggested?
Thanks

File: ckan/public/base/vendor/bootstrap/scss/_pagination.scss [hunk 1]
```diff
@@ -1,7 +1,100 @@
 .pagination {
-  display: flex;
+  display: inline-flex;
+  margin: 20px 0;
+  border-radius: 4px;
   @include list-unstyled();
 }
+.patination > li {
```
Comment (smotornyuk) [inline ckan/public/base/vendor/bootstrap/scss/_pagination.scss:7]: looks like you need `paGination`, not `paTination` here

**Comment by smotornyuk** [comment]
Yep. I mentioned a typo in a review comment. And you can make it a bit more SCSSish, like this:
```scss
.pagination > {
  li {
    display: inline;

    > {
      a, span {
        position: relative;
        float: left;
        padding: 6px 12px;
        margin-left: -1px;
        line-height: 1.42857143;
        color: #206b82;
        text-decoration: none;
        background-color: #fff;
        border: 1px solid #ddd;
      }

      a:hover, span:hover, a:focus, span:focus {
        z-index: 2;
        color: #113845;
        background-color: #eeeeee;
        border-color: #ddd;
      }
    }

    &:first-child > {
      a, span {
        margin-left: 0;
        border-top-left-radius: 4px;
        border-bottom-left-radius: 4px;
      }
    }

    &:last-child > {
      a, span {
        border-top-right-radius: 4px;
        border-bottom-right-radius: 4px;
      }
    }
  }

  .active > {
    a, span, a:hover, span:hover, a:focus, span:focus {
      z-index: 3;
      color: #fff;
      cursor: default;
      background-color: #206b82;
      border-color: #206b82;
    }
  }

  .disabled > {
    span {
      color: #777777;
      cursor: not-allowed;
      background-color: #fff;
      border-color: #ddd;

      &:hover, &:focus {
        color: #777777;
        cursor: not-allowed;
        background-color: #fff;
        border-color: #ddd;
      }
    }

    a {
      color: #777777;
      cursor: not-allowed;
      background-color: #fff;
      border-color: #ddd;

      &:hover, &:focus {
        color: #777777;
        cursor: not-allowed;
        background-color: #fff;
        border-color: #ddd;
      }
    }
  }
}

.pagination-lg > li {
  > {
    a, span {
      padding: 10px 16px;
      font-size: 18px;
      line-height: 1.3333333;
    }
  }

  &:first-child > {
    a, span {
      border-top-left-radius: 6px;
      border-bottom-left-radius: 6px;
    }
  }

  &:last-child > {
    a, span {
      border-top-right-radius: 6px;
      border-bottom-right-radius: 6px;
    }
  }
}

.pagination-sm > li {
  > {
    a, span {
      padding: 5px 10px;
      font-size: 12px;
      line-height: 1.5;
    }
  }

  &:first-child > {
    a, span {
      border-top-left-radius: 3px;
      border-bottom-left-radius: 3px;
    }
  }

  &:last-child > {
    a, span {
      border-top-right-radius: 3px;
      border-bottom-right-radius: 3px;
    }
  }
}
```

**Comment by TomeCirun** [comment]
screenshots:

image

image

**Comment by TomeCirun** [comment]
Thanks a lot, Sergey, as always, very helpful, I appreciate it.

**Comment by smotornyuk** [comment]
You're welcome:)
Verified

**Pull request merged**